src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -463,4 +464,28 @@ MemoryStoreStats CoreWorkerMemoryStore::GetMemoryStoreStatisticalData() {
   return item;
 }
 
+std::vector<ObjectID> CoreWorkerMemoryStore::GetAndDeletePlasmaObjectsOnRemovedNode(
```

Comment:
Oh, this answers my question before. We're now removing the entries when the node fails. Got it.